protobuf/v1/common.proto

```diff
@@ -1,10 +1,24 @@
 syntax = "proto3";
 
 package mayastor.v1;
-
+import "google/protobuf/timestamp.proto";
 // Protocol over which the bdev/replica/nexus etc will be shared.
 enum ShareProtocol {
   NONE = 0; // not exposed
   NVMF = 1; // NVMe over Fabrics (TCP)
   ISCSI = 2; // iSCSI
 }
+
+/// Parameter details of a single snapshot
+message ReplicaSnapshot {
+  string snapshot_uuid = 1; // uuid of the snapshot
+  string snapshot_name = 2; // name of the snapshot
+  uint64 snapshot_size = 3; // amount of bytes referenced by snapshot
+  uint64 num_clones = 4; // number of clones created from this snapshot
+  google.protobuf.Timestamp timestamp = 5; // snapshot creation_timestamp
+  string replica_uuid = 6; // uuid of the replica this snapshot is taken from
```

Comment:
I don't see the point of moving this to common, this is clearly a replica snapshot, not generic snapshot?